### PR TITLE
Remove unused hexdump module

### PR DESCRIPTION
**Pull request by data-man**
```nim
import hexdump

let s1 = "012\155\200"

echo dumpHex(s1[0].unsafeAddr, s1.len)
echo dumpHex(s1[0].unsafeAddr, s1.len, ascii = false)

let s2 = @[48'u8, 49, 50, 155, 200]
echo dumpHex(s2)
echo dumpHex(s2, ascii = false)
```

### CODE REVIEW

**Comment by data-man** [comment]
Oh, my fork is first! :smile:

File: ethp2p/hexdump.nim [hunk 1]
```diff
@@ -40,7 +40,7 @@ proc dumpHex*(pbytes: pointer, nbytes: int, items = 1, ascii = true): string =
     var k = 0
     while k < items:
       var ch = cast[ptr char](cast[uint](slider) + k.uint)[]
-      if ord(ch) > 31 and ord(ch) < 127: asciiText &= ch else: asciiText &= "."
+      if ascii or (ord(ch) > 31 and ord(ch) < 127): asciiText &= ch else: asciiText &= "."
```
Comment (yglukhov) [inline ethp2p/hexdump.nim]: This looks wrong. Why would you append non printable char to `asciiText`?

**Comment by yglukhov** [comment]
As a matter of fact I'm not sure if `dumpHex` should at all live in this repo. We already have byte-utils for such things.

**Comment by data-man** [comment]
Yes, I noticed that this function is not used in this repository.
However, there is a bug.

**Comment by yglukhov** [comment]
Let's just remove. No proc - no bug :)

**Comment by data-man** [comment]
Ok, removed.
Maybe move it to nim-ranges?

**Comment by cheatfate** [comment]
This not looks like proper removal... all tests got failed now.

**Comment by data-man** [comment]
Sorry, fixed.

**Comment by yglukhov** [comment]
Thanks!
> Maybe move it to nim-ranges?

I think https://github.com/status-im/nim-byteutils would be a better place for it. I'm not sure about your fix though. Why would anyone want nonprintable chars in the output.

**Comment by data-man** [comment]
> I think https://github.com/status-im/nim-byteutils would be a better place for it. 

I already adapted it for nim-ranges. :(

> Why would anyone want nonprintable chars in the output.

Because the output can be used in GUI.

**Comment by yglukhov** [comment]
@data-man, tbh i think such function would be more universal with openarrays interface. nim-ranges can convert to openarrays anyway.

**Comment by yglukhov** [comment]
1 of 5 procs depends on ranges and MemRange is easily converted to pointer/len. I honestly don't see how those belong to ranges. Also note that nim-ranges is mostly about data structures. What you're doing is an algorithm. E.g. if i as a user use seq[byte] all over the place and suddenly want to hexdump my data, i would first check `byteutils` package for that function. I'm not sure if i would check `ranges` at all...

**Comment by data-man** [comment]
Ok.
But in the ```byteutils``` there are no procs for working with pointers.

**Pull request merged**